include/sajson.h

```diff
@@ -44,6 +44,9 @@
 #define SAJSON_ALWAYS_INLINE __attribute__((always_inline))
 #define SAJSON_UNREACHABLE() __builtin_unreachable()
 #elif defined(_MSC_VER)
+#if (_MSC_VER <= 1800)
+#define snprintf _snprintf
```

Comment:
I don't think sajson should @define an un-namespaced symbol like this.

Would you mind changing uses of snprintf to use a new macro `SAJSON_snprintf` or something like that?